### DISCUSSION TITLE
Discord server

### DISCUSSION THREAD

**Discussion by pTinosq** [d2phap/ImageGlass#997] [General] [answered]
I'm just wondering if there's ever been any thought of starting a Discord server related to ImageGlass?

I think it'd be really beneficial as it'd reinforce a stronger community, allow for instant development feedback, allow for people to get mentioned when a new update is released and it could potentially act as marketing for the software.

**Comment by d2phap** [comment]
Hi @pTinosq 

Thanks for the suggestion. The problem is that we are lack of people, there are only me and Kevin, and we have our own work.
It's good to have a Discord server, but I worry that I cannot response fast enough. 😣

**Reply by pTinosq** [reply]
I understand where you come from, you've got 2.5k stars and almost 50 contributors. In regards to feeling like you can't respond fast enough, don't worry, that's the point of the community 😉, people who know how to answer a question will reply for you, you should also never feel like you _need_ to reply, you don't.

Still, if you'd rather not, I completely understand 👍 but if you need any help, I'm happy to chip in!

**Reply by d2phap** [reply]
Thanks! I will find some time to create and promote it on IG website. I will update here, too.

**Reply by pTinosq** [reply]
Fantastic, can't wait 😊

**Comment by d2phap** [comment, marked as answer]
Hi @pTinosq 
I have created a Discord server, welcome to join in at: http://discord.io/imageglass
